frameworks/PHP/symfony/symfony-franken.dockerfile

```diff
@@ -19,13 +19,13 @@ ADD . /symfony
 WORKDIR /symfony
 
 RUN mkdir -m 777 -p /symfony/var/cache/{dev,prod} /symfony/var/log
-RUN composer install --no-dev --no-scripts --quiet
+
+ENV COMPOSER_ALLOW_SUPERUSER=1
+RUN composer install --optimize-autoloader --classmap-authoritative --no-dev --quiet --no-scripts
```

Comment:
Running that before running `composer require runtime/frankenphp-symfony` is breaking it because `composer require` will perform a partial update and re-dump the autoload (and in that command, you don't enable the autoload optimizations).
There was a reason to dump the optimized autoloader after that before.